Storage/StorageTests/CoreData/CoreDataIterativeMigratorTests.swift

```diff
@@ -205,7 +232,17 @@ private extension CoreDataIterativeMigratorTests {
 }
 
 /// Helpers for the Core Data migration tests
-extension CoreDataIterativeMigratorTests {
+private extension CoreDataIterativeMigratorTests {
+
+    var documentsDirectory: URL {
+        let path = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true).last!
+        return URL(fileURLWithPath: path)
+    }
+
+    private func managedObjectModel(for modelName: String) throws -> NSManagedObjectModel {
```

Comment:
super nit: `private` isn't needed anymore

Reply:
🙈 Fixed in ab52c6841b5a35f14d8aac43eb4a16bc29449cd6. Thanks!